package.json

```diff
@@ -56,7 +58,6 @@
     "watchify": "^3.11.1"
   },
   "devDependencies": {
-    "@types/enzyme": "^3.10.5",
```

Comment:
I've removed this dependency temporarily to ease the initial adoption of JSDoc.

Reply:
Do we need to keep track of this?

Reply:
No. I may add it back in later as part of enabling checking for tests, but that's TBD at the moment.